bezalel001/Maintenance-Tracker
# Requests statuses added

**What does this PR do?**
Highlights the status of each request

**How should this be manually tested?**
When you navigate to the profile, you should see a list of requests with clear defined states/statuses of the requests
